apps/api/src/server/applications/application/documents/document.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * NSIP Document
+ *
+ * @typedef {Object} NsipDocumentPayload
```

Comment:
Is this the type defined in the json schema? Be nice if we didn't have to redefine. Will investigate.

Reply:
This will generate jsdoc, but I was hoping for some `import('my-schema.json`)` support!
https://www.npmjs.com/package/json-schema-to-jsdoc

Reply:
Looks good! Definitely agree, we're duplicating effort here - let's automate this if we can.

For this PR though, are you ok if I just merge it to get going?